components/_patterns/02-molecules/about-x/_about-x.scss

```diff
@@ -0,0 +1,20 @@
+$about-x--break: $medium; // This is a random breakpoint. Design should verify it.
```

Comment:
We have these frames up in space in Figma that I imagine we can anchor off of:

<img width="835" alt="Screen Shot 2019-09-10 at 9 31 56 AM" src="https://user-images.githubusercontent.com/1720093/64618244-e36ec080-d3ad-11e9-91cc-06c4c6be0e35.png">

If we find that these values aren't mapping well to responsive-izing the designs we're being given I'm sure design would be fine with us giving back suggestions; very few mobile designs have been completed.

Reply:
I'm going to update the site-wide breakpoints when I do the theme cleanup overhaul. So, I'm just going to resolve this for now.